### ISSUE TITLE
Spec 0.8: Add check indices are sorted in validate_indexed_attestation

### ISSUE THREAD

**Issue by prestonvanloon** [prysmaticlabs/prysm#2866]
Some context and discussion here: https://github.com/ethereum/eth2.0-specs/pull/1225

Adding this issue for tracking, we should see some new test failures in spectest since this _should_ be enforced. If it is not enforced in spec test, then we probably don't need it.

**Comment by prestonvanloon** [comment]
Blocked by https://github.com/ethereum/eth2.0-specs/pull/1238

**Comment by nisdas** [comment]
@prestonvanloon I think we can close this now, as its in the current spec test suite

**Comment by prestonvanloon** [comment]
Thanks @nisdas. It is indeed do. https://github.com/prysmaticlabs/prysm/blob/97b9176411113ff242ef2ec96a1f5c233d29f191/beacon-chain/core/blocks/block_operations.go#L716-L721

This can be optimized to O(n) runtime check. We don't need to actually store an array of numbers to ensure they are sorted. I'll follow up in another issue.
